devnet-sdk/system/chain_test.go

```diff
@@ -68,9 +67,64 @@ func TestChainFromDescriptor(t *testing.T) {
 				Address:    common.HexToAddress("0x1234567890123456789012345678901234567890"),
 			},
 		},
+		Addresses: descriptors.AddressMap{
```

Comment:
So my understanding here is that, on actual system there can be addresses which are not included in wallets right?

Reply:
Yes, for example contract addresses.